### PR TITLE
More work on the command `oq prepare_site_model`

### PR DESCRIPTION
**Pull request by micheles**
I have added documentation and tests. Short story:
```bash
$ oq prepare_site_model Exposure/Exposure_Res_Ecuador.csv,Exposure/Exposure_Res_Bolivia.csv Vs30/Ecuador.csv,Vs30/Bolivia.csv --grid-spacing=10
```
will produce a sites.csv file with the right vs30 values.

### CODE REVIEW

**Comment by micheles** [comment]
Post mortem review

File: doc/oq-commands.md [hunk 1]
```diff
@@ -182,3 +182,58 @@ Be warned that for large calculations the extraction will likely be slower
...
+```
+
+The command work in two modes: with non-gridded exposures (the
+default) and with gridded exposures. In the first case the assets are
+aggregated in unique locations and for each location the vs30 coming
+from the closest vs30 record is taken. In the second case, i.e. when a
+grid spacing parameter is passed, a grid containing of all the
+exposure is built and the points with assets are associated to the
+vs30 records. The maximum association distance is given
+by the grid spacing multiply by the square root of 2, so by enlarging
```
Comment (ptormene) [inline doc/oq-commands.md]: multiply -> multiplied

File: doc/oq-commands.md [hunk 2]
```diff
@@ -182,3 +182,58 @@ Be warned that for large calculations the extraction will likely be slower
...
+the grid spacing you can avoid discarding the assets.
+
+In large risk calculation one wants to *use the gridded mode always* because:
```
Comment (ptormene) [inline doc/oq-commands.md]: calculation -> calculations

File: openquake/commands/prepare_site_model.py [hunk 3]
```diff
@@ -32,52 +32,65 @@
 
 
 F32 = numpy.float32
+SQRT2 = 1.414
```
Comment (ptormene) [inline openquake/commands/prepare_site_model.py]: Is this approximation ok?

**Comment by micheles** [comment]
Yes, more than enough.

File: doc/oq-commands.md (hunk 1 above)
Comment (micheles) [reply doc/oq-commands.md]: Done

**Pull request merged**